integration-tests/jest/jest-utils.js

```diff
@@ -73,26 +78,30 @@ const flakeShaker = ({ fn, name }) => {
       const { taskFn, taskParams, runId } = data
       try {
         const result = await taskFn({ context, page, ...taskParams })
-        rawConsole.log(`Test number ${runId} passed`)
+        process.stdout.write('.')
```

Comment:
When I run the tests I am not seeing the dot printed out. If I add a newline, then I do see the dots get printed.
I tried changing this to ```process.stdout.write(`Reached test ${runId}`)``` to reduce log noise, but some tests console out other information which ends up overwriting this line, resulting in it not getting seen.

Is the intention to reduce the log noise?

I originally added this logging to let the user know the tests are running and give them a sense of the progress. I do like the idea of removing all of this noise and updating the same line with the progress.

We could could capture `stdout` to a variable and redirect `process.stdout.write` to a log while the tests are running and then restore `stdout` when the tests are done. I think this would significantly reduce command line noise.

Reply:
Yeah, I was hoping to reduce the log noise, and just have the dots indicate something was happening - Let me see if I can get it a little more consistent. I was erasing them  after the cluster.task await finished, but that might be the wrong time?

Reply:
Found the problem, we also needed to be resetting the cursor for stdout! updated the messaging to give you a percentage competed now instead, cause I think that's probably better than adding dots too